secrets/secrets-kubernetes/Dockerfile

```
@@ -0,0 +1,20 @@
+FROM golang:1.22 AS builder
+ARG TARGETOS
+ARG TARGETARCH
+
+WORKDIR /workspace
+COPY go.mod go.mod
+COPY go.sum go.sum
+RUN go mod download
+
+COPY *.go .
+COPY pkg/ pkg/
+
+RUN CGO_ENABLED=0 GOOS=${TARGETOS:-linux} GOARCH=${TARGETARCH:-amd64} go build -a -o secrets-kubernetes
+
+FROM gcr.io/distroless/static:nonroot
```

I don't know what the plain `static` image defaults to, so it might be worth being explicit:
```suggestion
FROM gcr.io/distroless/static-debian12:nonroot
```

---

As an aside, it might be nice to also create a separate `debug` image based on the `:debug` tag so that if there was a need to do debugging, it would be possible to get use a version of the image with a minimal busybox setup: https://github.com/wasmCloud/wasmCloud-contrib/blob/main/secrets/secrets-vault/Dockerfile#L15-L21

---

yas. getting one too.